solidity/dashboard/src/components/LiquidityRewardCard.jsx

```diff
@@ -0,0 +1,64 @@
+import React from "react"
+import DoubleIcon from "./DoubleIcon"
+import * as Icons from "./Icons"
+import { SubmitButton } from "./Button"
+import Card from "./Card"
+
+const LiquidityRewardCard = ({ title, MainIcon, SecondaryIcon, viewPoolLink }) => {
+  return (
+    <Card className={"tile"}>
+      <div className={"liquidity__card-title-section"}>
+        <DoubleIcon
+          MainIcon={MainIcon}
+          SecondaryIcon={SecondaryIcon}
+          className={`liquidity__double-icon-container`}
+        />
+        <h2 className={"h2--alt text-grey-70"}>{title}</h2>
+      </div>
+      <div className={"liquidity-card-subtitle-section"}>
+        <span className="text-grey-40">Uniswap Pool&nbsp;</span>
```

Comment:
Based on the Figma views it should be `h4` tag. So I think we can drop `<div className={"liquidity-card-subtitle-section"}>` and:
 
 ```
<h4 className="liquidity-card__subtitle text-grey-40">
 Uniswap Pool&nbsp;
 <a href={viewPoolLink} className="arrow-link text-small">
  View pool
 </a>
</h4>
 ```

Reply:
Fixed in https://github.com/keep-network/keep-core/pull/2216/commits/bcb0728b0a7808362814406b1cbb02beca04ea7d